doc/exampleconfigs/mediator-example.yaml

```diff
@@ -58,9 +60,12 @@ pcapcompress: 1
 # then you will need to provide these options regardless (and therefore will
 # have to accept encrypted inter-component communication).
 #
-#tlscert: /etc/openli/openli-mediator-crt.pem
-#tlskey: /etc/openli/openli-mediator-key.pem
-#tlsca: /etc/openli/openli-ca-crt.pem
+# The TLS certificates and keys can be generated using PQC cryptography algorithms, based on OQS-Provider list of algorithms. 
+# As well, is required to modify the "Openssl.cnf" file to add the desired QS Key Exchange Methods (KEM) desired.
+#
+tlscert: /etc/openli/openli-mediator-crt.pem
```

Comment:
Please revert to TLS being disabled by default in the example config files, as per earlier comment on the collector example config file.